Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardInputViewModel.swift

```diff
@@ -238,6 +253,23 @@ class CreditCardInputViewModel: ObservableObject {
                                   completion: completion)
     }
 
+    private func dismissAndRemoveCreditCard() {
+        guard let currentCreditCard = creditCard,
+              !currentCreditCard.guid.isEmpty else {
+            return
+        }
+        autofill.deleteCreditCard(id: currentCreditCard.guid) {
+            status, error in
+            guard let error = error, status == true else {
```

Comment:
nit: we could `remove == true`
```suggestion
            guard let error = error, status else {
```

Reply:
Oops, cleaned